<ui version="4.0" >
 <class>ReservationEditor</class>
 <widget class="QWidget" name="ReservationEditor" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>434</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>ReservationEditor</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="topMargin" >
    <number>0</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <property name="bottomMargin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Fournissez des informations pour un client reservé</string>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" >
     <item row="0" column="0" colspan="2" >
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Nom de la réservation :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2" >
      <widget class="QLineEdit" name="lineName" />
     </item>
     <item row="1" column="0" colspan="2" >
      <widget class="QLabel" name="label_4" >
       <property name="text" >
        <string>Adresse MAC :</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2" >
      <widget class="QLineEdit" name="lineMacAddress" >
       <property name="inputMask" >
        <string>HH:HH:HH:HH:HH:HH;_</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2" >
      <widget class="QLabel" name="label_6" >
       <property name="text" >
        <string>Type matériel :</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2" >
      <widget class="QComboBox" name="comboMacType" >
       <item>
        <property name="text" >
         <string>Ethernet</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>FDDI</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Token-Ring</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0" colspan="2" >
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>Adresse IP :</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2" >
      <widget class="QLineEdit" name="lineIPAddress" >
       <property name="inputMask" >
        <string>000.000.000.000;_</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2" >
      <widget class="QLabel" name="label_5" >
       <property name="text" >
        <string>Déscription :</string>
       </property>
      </widget>
     </item>
     <item rowspan="2" row="4" column="2" >
      <widget class="QTextEdit" name="textDescription" />
     </item>
     <item row="5" column="0" colspan="2" >
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
